pkg/config/model/config_overrides.go

```diff
@@ -3,9 +3,12 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
-package config
+package model
```

Comment:
The name `model` makes sense when you are seeing the folder structure, e.g. `pkg/config/model`.
However when used in the code you end up with `model.XXX`, which does not make any sense to the reader.

Given that we're going to remove the aliases, I hope very soon, I think it's important to find a better name.

Reply:
We iterated on name few times. Initial proposal was `pkg/conf`. We settled on `pkg/config/model` as middle ground. Do you any suggestions for name ?

Reply:
We can do `pkg/config/config` as we're going to remove everything in `pkg/config` anyway.

Reply:
I think `pkg/config/config` will confuse the developer because of aliases; both types would be present in two packages with the same name.  I don't have clear picture what would be the final state of `pkg/config` . We can revisit this in future .

Reply:
Discussed offline. 
1. We will be using import aliases like  `configModel "pkg/config/model" ` to denote that it is config model . 
2. We will minimize the code that refers `pkg/config/model` and try to use config component throughout the code. 
We will use pkg/config/model in 4 places 
1.  To init config component 
2. To create a mock config for tests 
3. In the DD exporter.
4. Non-comp code that needs to be converted to go module would use pkg/config/model